icons/lightbulb.svg

```diff
@@ -11,5 +11,5 @@
 >
   <line x1="9" x2="15" y1="18" y2="18" />
   <line x1="10" x2="14" y1="22" y2="22" />
-  <path d="M15.09 14c.18-.98.65-1.74 1.41-2.5A4.65 4.65 0 0 0 18 8 6 6 0 0 0 6 8c0 1 .23 2.23 1.5 3.5A4.61 4.61 0 0 1 8.91 14" />
+  <path d="M15 14c0-1.2.7-1.7 1.5-2.5 1-.9 1.5-2.2 1.5-3.5 0-3.3-2.7-6-6-6S6 4.7 6 8c0 1 .2 2.2 1.5 3.5.7.7 1.5 1.3 1.5 2.5" />
```

Comment:
```suggestion
  <path d="M10 22h4"/>
  <path d="M9 18h6"/>
  <path d="M15 14a3.34 3.34 0 0 1 1.5-2.5 5.8 5.8 0 1 0-9 0A3.5 3.5 0 0 1 9 14"/>
```

Reply:
Hmm, these changes result in some less than pretty curvature, and non pixel prefect dimensions of 11.6002x11.9594px…

<img width="638" alt="Untitled" src="https://user-images.githubusercontent.com/7797479/232446173-73f6c982-9d04-49ba-80ce-4d239e10ba3d.png">

<img width="56" alt="Untitled" src="https://user-images.githubusercontent.com/7797479/232446077-ac77edf8-85ad-43fe-9307-e98d87813150.png">

Reply:
Oh, you are right.

This should be better:
```suggestion
  <path d="M10 22h4" />
  <path d="M9 18h6" />
  <path d="M15 14c0-1.2.7-1.7 1.5-2.5a5.8 5.8 0 1 0-9 0c.7.7 1.5 1.3 1.5 2.5" />
```

Reply:
Does it matter that this makes the bulb 11.6002x11.9594px, as opposed to 12x12px…?